pages/index.js

```diff
@@ -31,18 +35,37 @@ export default function Home() {
         <title>Quiz Demo</title>
         <link rel="icon" href="/favicon.ico" />
       </Head>
-      <main className="flex justify-center min-h-screen w-full md:py-32 py-0">
+      <main className="flex flex-col items-center justify-center min-h-screen w-full md:py-32 py-0">
         {state.matches('initializing') ? (
           'loading...'
         ) : (
-          <QuestionToShow
-            state={state}
-            handleSubmit={handleSubmit}
-            question={currentQuestion}
-            handleContinue={handleContinue}
-            currentAnswer={currentAnswer}
-            isDisabled={isDisabled}
-          />
+          <>
+            {map(
+              filter(state.context.answers, (a) => a.id !== currentQuestion.id),
+              (answer) => (
+                <AnsweredQuestionToShow
```

Comment:
array of answers is essentially a history. answer is a question with value representing user's answer.